Makefile

```
@@ -722,12 +722,10 @@ package-repo-bundle: tools prep-package-tools ## Build tar bundles for package r
 	cd hack/packages/package-tools && $(GO) run main.go repo-bundle generate --repository=$(PACKAGE_REPOSITORY) --registry=$(OCI_REGISTRY)/packages/$(PACKAGE_REPOSITORY) --version=$(REPO_BUNDLE_VERSION) --package-values-file=$(PACKAGE_VALUES_FILE) --sub-version=$(REPO_BUNDLE_SUB_VERSION)
 
 .PHONY: push-package-bundles
-ifeq (push-package-bundles, $(firstword $(MAKECMDGOALS))) ## Allow passing in arguments to the make command
-  push_package_bundles_args := $(wordlist 2, $(words $(MAKECMDGOALS)), $(MAKECMDGOALS))
-  $(eval $(push_package_bundles_args):;@:)
-endif
-push-package-bundles: tools prep-package-tools ## Push specified package bundle(s) in a package repository
-	cd hack/packages/package-tools && $(GO) run main.go package-bundle push --repository=$(PACKAGE_REPOSITORY) --registry=$(OCI_REGISTRY)/packages/$(PACKAGE_REPOSITORY) --version=$(BUILD_VERSION) --sub-version=$(PACKAGE_SUB_VERSION) $(push_package_bundles_args)
+push-package-bundles: tools prep-package-tools ## Push specified package bundle(s) in a package repository. 
+## Specified package bundles must be set to the PACKAGE_BUNDLES environment variable as comma-separated values 
+## and must not contain spaces. Example: PACKAGE_BUNDLES=featuregates,core-management-plugins
+	cd hack/packages/package-tools && $(GO) run main.go package-bundle push --repository=$(PACKAGE_REPOSITORY) --registry=$(OCI_REGISTRY)/packages/$(PACKAGE_REPOSITORY) --version=$(BUILD_VERSION) --sub-version=$(PACKAGE_SUB_VERSION) $(PACKAGE_BUNDLES)
```

```suggestion
	cd hack/packages/package-tools && $(GO) run main.go package-bundle push $(PACKAGE_BUNDLES) --repository=$(PACKAGE_REPOSITORY) --registry=$(OCI_REGISTRY)/packages/$(PACKAGE_REPOSITORY) --version=$(BUILD_VERSION) --sub-version=$(PACKAGE_SUB_VERSION)
```
Can you make the above change just to be more clear that `$(PACKAGE_BUNDLES)` is the argument to the go program? having the argument beside `sub-version` flag can be confusing when the `sub-version` flag value is empty ex :
`cd hack/packages/package-tools && go run main.go package-bundle push --repository=management --registry=quay.io/hyayiv --version=v0.19.0-dev --sub-version= featuregates`

---

I applied your suggestion.